packages/cursorless-engine/src/actions/FollowLink.ts

```diff
@@ -24,11 +29,12 @@ export default class FollowLink implements SimpleAction {
       .openLink(target.contentRange);
```

Comment:
You should pass an options object here to indicate link should be opened aside, and then handle that [here](https://github.com/cursorless-dev/cursorless/blob/main/packages/cursorless-vscode/src/ide/vscode/VscodeOpenLink.ts#L66) by passing an options object there to indicate to open aside